packages/vuetify/src/components/VNavigationDrawer/touch.ts

```diff
@@ -0,0 +1,175 @@
+import { useVelocity } from '@/composables/touch'
```

Comment:
should this be a reusable composable?

Reply:
Everything in here is specific to drawers